server/server_test.go

```diff
@@ -386,3 +386,34 @@ func TestNilMonitoringPort(t *testing.T) {
 		t.Fatal("HttpAddr should be nil.")
 	}
 }
+
+type DummyAuth struct {
+	count int
+}
+
+func (d *DummyAuth) Check(c ClientAuth) bool {
+	d.count++
```

Comment:
Would remove that and make it return false always, or if you want to make test more sophisticated, you could check `c.GetOpts().Username` and return value based on that.